test/LibraryManager.IntegrationTest/LibmanCompletionTests.cs

```diff
@@ -157,5 +157,22 @@ public void LibmanCompletion_LibraryForUnpkg()
             Editor.KeyboardCommands.Type("\"library\":");
             LibmanTestsUtility.WaitForCompletionEntries(Editor, expectedCompletionEntries, caseInsensitive: true, timeout: 5000);
         }
+
+        [TestMethod]
+        public void LibmanCompletion_LoadingNotInserted()
+        {
+            _libManConfig.Open();
+            string expectedText = "      \"library\": ";
+
+            Editor.Caret.MoveToExpression("\"libraries\"");
+            Editor.Caret.MoveDown(2);
+            Editor.KeyboardCommands.Type("\"provider\": \"cdnjs\",");
+            Editor.KeyboardCommands.Enter();
+            Editor.KeyboardCommands.Type("\"library\":");
+            Editor.KeyboardCommands.Type("j\t");
+
```

Comment:
If you run this with other tests, do you hit Loading here or does it go straight to cached data.